### DISCUSSION TITLE
What is happening if VPN falls down ?

### DISCUSSION THREAD

**Discussion by manitou153** [qdm12/gluetun#1374] [Q&A] [answered]
Hi everyone
I am not experienced enough to do some test on my gluetun container.
I'd like to know what is happening if my provider VPN server falls down for a while. Does Gluetun stop all the traffic egress and ingress and does he attempt to reconnect automatically ?  I think that the container have a killswitch fonction but I don't know how it works.

Thank you for your help.

**Comment by qdm12** [comment, marked as answer]
See https://github.com/qdm12/gluetun/wiki/Firewall 😉

**Reply by manitou153** [reply]
Thank you for this link ! I understand better how it works now.
